### PR TITLE
Weekly meeting, 2018-04-05

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2018-04-05-coordination.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
+---
+title: "HSF Weekly Meeting #132, 4 April, 2018"
```
Comment (eduardo-rodrigues) [inline organization/_posts/2018-04-05-coordination.md]: 5 April ;-).

File: organization/_posts/2018-04-05-coordination.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+===========================
+-   Gather important items that we should be tracking after the workshop
+    (see workshop [live
+    notes](https://docs.google.com/document/d/1QSkvwRK_2HENuxYXcs9Op1dTUK824KddQ1Tfan-P0WU/edit?usp=sharing))
+    -   ### General Points
+        -   We need a newsletter!
+            -   Graeme to follow up
+        -   Benedikt will give a WS summary in next week's EP-SFT meeting.
+    -   ### Common Data Management and Data Lakes
+        -   WLCG lead here
```
Comment (eduardo-rodrigues) [inline organization/_posts/2018-04-05-coordination.md]: Not sure whether you care too much but there are several items throughout the minutes which do not finish with a dot.

File: organization/_posts/2018-04-05-coordination.md (hunk 1 above)
Comment (graeme-a-stewart) [reply organization/_posts/2018-04-05-coordination.md]: !

File: organization/_posts/2018-04-05-coordination.md (hunk 2 above)
Comment (graeme-a-stewart) [reply organization/_posts/2018-04-05-coordination.md]: Not too much, but I put most of them in the last commit.

**Pull request merged**